doc/nrf/releases/release-notes-1.6.0.rst

```diff
@@ -0,0 +1,471 @@
+
+  * :ref:`aws_fota_sample` sample:
+
+    * Removed nRF Connect for Cloud support code, because ``fota_v1`` is no longer supported in nRF Connect for Cloud.
+    * Removed provisioning using :ref:`modem_key_mgmt` and :file:`certificates.h`, because this is not the recommended way of provisioning private certificates.
+    * Renamed the following Kconfig options:
+
+      * ``CONFIG_CLOUD_CERT_SEC_TAG`` renamed to :option:`CONFIG_CERT_SEC_TAG`.
+      * ``CONFIG_USE_CLOUD_CLIENT_ID`` renamed to :option:`CONFIG_USE_CUSTOM_CLIENT_ID`.
+      * ``CONFIG_CLOUD_CLIENT_ID`` renamed to ``CONFIG_CLIENT_ID``.
+      * ``CONFIG_NRF_CLOUD_CLIENT_ID_PREFIX`` renamed to ``CONFIG_CLIENT_ID_PREFIX``.
+
+  * :ref:`lib_aws_fota` library:
+
+    * Updated internal state handling and fault tolerance.
+
+  * :ref:`sms_readme` updated to enable sending of sms and decoding of received SMS payload.
+
+  * :ref:`liblwm2m_carrier_readme` library - Updated to v0.20.1. See :ref:`liblwm2m_carrier_changelog` for details.
+
+  * :ref:`lib_lwm2m_client_utils` created from common parts of :ref:`lwm2m_client`. This module can be reused to add common objects to LwM2M applications.
+
+nRF5
+====
+
+Bluetooth LE
+------------
+
+* Added:
+
+  * Support for serialization of the :ref:`zephyr:bt_gap` and the :ref:`zephyr:bluetooth_connection_mgmt` API.
+  * :ref:`ble_rpc_host` sample that enables support for serialization of the :ref:`zephyr:bt_gap` and the :ref:`zephyr:bluetooth_connection_mgmt`.
+
+* Updated:
+
+  * :ref:`shell_bt_nus` sample - Fixed an issue where shell transport did not display the initial shell prompt ``uart:~$`` on the remote terminal.
+
+Matter (Project CHIP)
+---------------------
+
+* Project CHIP has been officially renamed to `Matter`_.
+* Added:
+
+  * New user guide about :ref:`ug_matter_configuring`.
+
+* Updated:
+
+  * Renamed occurrences of Project CHIP to Matter.
+
+Thread
+------
+
+* Added support for the following Thread 1.2 features:
+
+  * Domain Unicast Addresses
+  * Multicast Listener Registration
+  * Backbone Router (Thread Network side only)
+  * Link Metrics
+  * Coordinated Sampled Listening (CSL)
+
+  Link Metrics and CSL are supported for nRF52 Series devices.
+
+* NCP sample renamed to Co-processor, with added support for RCP architecture.
+
+Zigbee
+------
+
+* Added the v0.9.5 version of the `ZBOSS NCP Host`_ package that includes a simple gateway application.
+* Updated:
+
+  * Reworked the :ref:`NCP sample <zigbee_ncp_sample>` to work with the simple gateway application.
+  * Moved the `NCP Host documentation`_ from the `ZBOSS NCP Host`_ package to the same location as the `external ZBOSS development guide and API documentation`_.
+
+nRF Desktop
+-----------
+
+* Fixed issues with boot reports on the USB backend.
+* Adapted the application to use common modules from the :ref:`lib_caf` library.
+* Fixed minor bugs.
+
+Common
+======
+
+The following changes are relevant for all device families.
+
+Bluetooth mesh
```

Comment:
From #4856  @Balaklaka  @trond-snekvik

Reply:
@b-gent I believe Bluetooth Mesh should go in the nRF5 section, between Bluetooth LE and Matter. It does not run on nRF91.

Reply:
Moved, thanks